### DISCUSSION TITLE
Rtab map is very slow on very powerful laptop

### DISCUSSION THREAD

**Discussion by damavand1** [introlab/rtabmap#1125] [General]
hi

my laptop is 
intel i9
3060 nvidia 6 gb
16 gb ram

on ubuntu 22.04

and i'm using xbox kinect 360

when i'm scanning my room, many times screen be in red and very very slow
and can't continue anymore

it's good to know i move my hand very slow and not fast

now, my question is  whats the problem?

every things are in default config
ros 2 humble
and
rtabmap-ros

and i run rtabmap using "Rtabmap" command

**Comment by matlabbe** [comment]
There are many reasons why that could happen, look at section "Red screens" on this page: https://github.com/introlab/rtabmap/wiki/Kinect-mapping#lost-odometry-red-screens
